setup/set_up_mac.md

```diff
@@ -53,64 +54,59 @@ You should see a list of usage options.
-**NOTE**: If the response is �The document has moved�, then curl was successfully installed, but the url is not correct. According to your geography, you need to change the extension.
+**NOTE**: If the response is �The document has moved�, then curl was successfully installed, but the url is not correct. According to your geography, you need to change the extension.
+3. If you do not have curl installed, you can get it from the Mac App Store (http://macappstore.org/curl), or you can find various version binaries and source code at https://curl.haxx.se/download.html .
 
-### Step 7: Install node.js
-1.Type  
+### Step 8: Install node.js
+1. Type  
 	`brew install node `
 2. Verify the installation:  
-`node �v`  
+`node -v`  
 The response is the version number (for example, v4.2.6).
-4. Type  
+3. Type  
 `npm -v`  
 Again, the response is the version number.
 
-### Step 8: Install API Connect
-Use npm to install API Connect:  
-1. Type ` npm install �g apiconnect`  
-NOTE: This takes some time. You see some warnings about deprecations; you can ignore them.
-2. Verify the installation by typing  
-`apic �v`  
-You are asked to accept the license, and to allow usage information to be collected. Take the default answers for these (�Yes� and �No�). You then see the version number.
-
 ### Step 9: Install Docker
 1. In a browser, open   
 [https://docs.docker.com/docker-for-mac/](https://docs.docker.com/docker-for-mac/)
-2. Scroll down and click the button *Get Docker for Mac (stable)*
-3. Open the downloaded file.
-4. A window opens with two icons. Drag the left *Docker.app* icon onto the right *Applications* icon.
-5. In the *Applications* directory, double-click Docker.app.
-6. Verify that the whale icon is added to the status bar at the top of the screen.
-7. Click the icon to verify that Docker is running.
+2. In the left pane, click `Install Docker for Mac`.
+3. Click the button `Download from Docker Store`.
+4. Follow the instructions on the web page to install Docker and make sure it is running.
+
+### Step 10: Install the IBM Cloud Bluemix CLI
+1. In a browser, go to https://console.bluemix.net/docs/cli/reference/bluemix_cli/get_started.html
+2. Click the installer for MacOS and follow the instructions to install the Bluemix CLI.
+3. Type `bx -v` to return the version number.
 
-### Step 10: Install cloudfoundry CLI
-You install cloudfoundry using a package manager.
-1. In a Terminal, type  
-`$ brew tap cloudfoundry/tap`
-2. Install:  
-`$ brew install cf-cli`
-3. In order to verify the installation, type  
-`cf �v`  
-The response shows the version number.
+### Step 11: Install the IBM Container Service plugins
+There are two plugins associated with the IBM Cloud Container Service that you will need for the labs. They are the plugin for the container service itself (`bx cs` commands) and the plugin for the container registry (`bx cr` commands).
+1. Type  
+`bx plugin install container-service -r Bluemix`
```

Comment:
I think lines 102 and 103 should be separate commands so might need a `.br` or something?

Reply:
OK. Will do. Sometimes or always you seem to need a <br> between 2 commands.